### PR TITLE
TPC: Update of Tracks task

### PR DESCRIPTION
**Pull request by stheckel**
o Add publishing of ratio plots (neg./pos. tracks)
o Add calling of new function processEndOfCycle

### CODE REVIEW

**Comment by stheckel** [comment]
This commits needs the following O2 PR to be merged first:
[https://github.com/AliceO2Group/AliceO2/pull/6824](https://github.com/AliceO2Group/AliceO2/pull/6824)

File: Modules/TPC/src/Tracks.cxx [hunk 1]
```diff
@@ -49,6 +49,11 @@ void Tracks::initialize(o2::framework::InitContext& /*ctx*/)
     getObjectsManager()->startPublishing(&hist2);
     getObjectsManager()->addMetadata(hist2.GetName(), "custom", "42");
   }
+
+  for (auto& histR : mQCTracks.getHistogramRatios1D()) {
+    getObjectsManager()->startPublishing(&histR);
+    getObjectsManager()->addMetadata(histR.GetName(), "custom", "51");
```
Comment (wiechula) [inline Modules/TPC/src/Tracks.cxx:55]: Isn't this a leftover from some example? I guess it is not needed. Perhaps this should also be removed everywhere else.
```suggestion
```

**Review by wiechula — CHANGES_REQUESTED** [review]

File: Modules/TPC/src/Tracks.cxx (hunk 1 above)
Comment (stheckel) [reply Modules/TPC/src/Tracks.cxx]: Yes, I agree, this is not needed.

**Review by wiechula — APPROVED** [review]

**Comment by stheckel** [comment]
Thanks, Jens, for approving this one and the corresponding O2 commit. The latter is merged, so I remove the draft status here.

**Pull request merged**